tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Cacti
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+- Cacti is doing great, there is steady interest in the project as interoperability is becoming more and more important to organizations wishing to deploy distributed ledger technologies.
+- There was a significant increase in new contributors this last quarter according to LFX (75%) and the pair programming sessions get more attendance than ever.
+- The project repository was successfully moved to [its own repository](https://github.com/hyperledger-cacti) following the move from the Hyperledger Foundation to Linux Foundation's Decentralized Trust. Cacti was one of the first projects on which this experiment was carried out, and it was successful thanks to the LFDT administrators.
+- The mentorship projects are progressing nicely, with solid code contributions from the mentees, including Cactus-Weaver integration.
+- Work on keeping Cacti up-to-date with the IETF SATP interoperability protocol standard is going well.
+  * Cacti maintainers continue to be actively involved in the IETF SATP Working Group, co-authoring protocol specifications.
+- We are not doing too badly on the [OpenSSF Scorecard Metrics](https://scorecard.dev/viewer/?uri=github.com%2Fhyperledger-cacti%2Fcacti) with a present score (as of the time of this file's commit) of 6.3, but there is a lot of scope to improve.
+
+# Questions/Issues for the TAC
+
+No questions at the moment, thank you.
+
+# Releases
+
+Issued releases:
+- v2.0.0-rc.4
+- v2.0.0-rc.5
+- v2.0.0-rc.6
+- v2.0.0-rc.7
+- v2.0.0
+
+Caveat: We are still working on the publishing workflows that broke down after our latest org migration.
```

Comment:
Q: Would this impact existing consumers of cacti libraries?

Reply:
@arsulegai Not on a technical level, no, as-in there should be no breakage from this at all. I can see people being disappointed though when they see the release tags in the git commit log but cannot locate the artifacts on npm (which is a temporary setback we hope to overcome soon)